client/src/js/services/ModalService.js

```diff
@@ -69,5 +74,80 @@ function ModalService(Modal) {
     });
   }
 
+  /**
+   * Select cashbox modal
+   */
+  function openSelectCashbox(request) {
+    var cashboxIsSet = request.cashbox && request.cashbox.id;
+    var instance = Modal.open({
+      templateUrl : 'partials/cash/modals/selectCashbox.modal.html',
+      controller  : 'SelectCashboxModalController',
+      controllerAs: '$ctrl',
+      size        : 'md',
```

Comment:
Since most of these modals use the same defaults, one optimization you could is to put them in an object at the top of the service, then `angular.extend()` the modal defaults with these options.  Like this:
```js

var defaults = {
  size : 'md',
  backdrop : 'static',
  animation : false
};

// later, in each modal definition...
function openSelectCashbox(request) {
  var params = angular.extend(defaults, {
    templateUrl : 'partials/cash/modals/selectCashbox.modal.html',
    controller : 'SelectCashboxController as $ctrl'
  });

  // etc..
  Modal.open(params).then(...);
};
```